src/ceph-volume/ceph_volume/devices/lvm/listing.py

```diff
@@ -17,7 +17,7 @@
 
 osd_device_header_template = """
 
-  [{type: >4}]    {path}
+  {type: <11} {path}
```

Comment:
would you help me understand why this change is needed? since it is all in one commit I can't tell

Reply:
If the characters of the are more than 4, the [journal]'s, [block]'s and [  db]'s path values are not aligned properly. I guess I should change command outputs of some tests, because the spacing is changed.

Reply:
I think I am missing something here. The output example in the docs have more than four chars and I don't see them misaligned: http://docs.ceph.com/docs/master/ceph-volume/lvm/list/#full-reporting

Can you clarify where is the difference?

Reply:
While `journal uuid`, `osd id`, `cluster fsid` values are aligned, the `[data]` and `[journal]` are not.
![screenshot from 2018-05-17 13-40-00](https://user-images.githubusercontent.com/1884476/40175429-5da2a43a-59d8-11e8-9a87-737036a5d0e0.png)